model/clusters_mgmt/v1/flavour_type.model

```diff
@@ -41,4 +38,15 @@ class Flavour {
 	//
 	// These can be overriden specifying in the cluster itself a different set of settings.
 	Network Network
+
+	// AWS default instance type for the master volume.
+	MasterGroupInstanceType String
```

Comment:
Remove the `Group` part: it doesn't mean anything.

Reply:
fixed